### ISSUE TITLE
call Lint.jl

### ISSUE THREAD

**Issue by jeff-regier** [jeff-regier/Celeste.jl#257]
Once Lint.jl supports Julia 0.5, add something like

```
@test isempty(lintpkg("Celeste"))
```

to our unit tests, and fix all the issues.

**Comment by jeff-regier** [comment]
Lint.jl is working  with Julia 0.5 now: tonyhffong/Lint.jl#146

**Comment by rgiordan** [comment]
I ran `Lint.lintpkg` on Celeste, and (like Jeff) I didn't find the output very useful.  I was hoping for spacing violations, notification of undocumented functions, incorrect cases, non-spaced commas or binary operators, &c.  But instead I got
- A bunch of undeclared variable warnings (since we use a lot of global constants, or `include` .jl files in Modules for organization. 
- Some not-too-important suggestions for local types (e.g. to declare the interating variable in a for loop local)
- Some good but not critical advice about not masking things defined in Base 
- Some erroneous errors (like saying an array argument in a modifying function was unused when it was assigned to)

So I'm not sure how useful `Lint` is as of right now.
